modules/ob-configuring-pods-in-build.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is included in the following assembly:
+//
+// * configuring/configuring-openshift-builds.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-configuring-pods-in-build_{context}"]
+= Pod configuration
+
+You can use the following optional `Build` CR fields to configure {builds-shortname} pods:
```

Comment:
See my earlier comment about `{builds-shortname} pods` vs. `Build pods`. Should the phrase be `Builds pods`, `Build pods`, or `build pods`?